client/app/components/queries/ScheduleDialog.test.js

```diff
@@ -78,7 +71,7 @@ describe('ScheduleDialog', () => {
       const [wrapper] = getWrapper({
         interval: 1209600,
         time: '22:15',
-        day_of_week: 2,
+        day_of_week: 'Monday',
```

Comment:
Setting the prop types actually alerted a mistake in my test! This deserves the label :P

Reply:
I suspect you did this on purpose, just so you can tag another pull request :P